src/test/store/receive-profile.test.js

```diff
@@ -478,24 +478,23 @@ describe('actions/receive-profile', function() {
   });
 
   describe('retrieveProfileFromStore', function() {
-    const fetch403Response = { ok: false, status: 403 };
-    const fetch500Response = { ok: false, status: 500 };
-    const fetch200Response = {
+    const fetch403Response = (({ ok: false, status: 403 }: any): Response);
+    const fetch500Response = (({ ok: false, status: 500 }: any): Response);
```

Comment:
I think we can use a real `Response` object here:
```
const fetch403Response = new Response(null, { status: 403 });
const fetch500Response = new Response(null, { status: 500 });
```

Reply:
Ah no, `Response` doesn't exist in node... :sad:
Then please add a comment explaining that. (we like to explain why we use `any` when this happens)